controllers/start_test_env.go

```diff
@@ -122,6 +122,9 @@ func setupTestEnv() {
 		GitHubTokenClient:  mockGhTokenClient,
 		DevfileRegistryURL: cdqanalysis.DevfileStageRegistryEndpoint, // Use the staging devfile registry for tests
 		AppFS:              ioutils.NewMemoryFilesystem(),
+		Config:             cfg,
+		RunKubernetesJob:   true,
+		CdqAnalysisImage:   "quay.io/redhat-appstudio/cdq-analysis:next",
```

Comment:
Should we update the readme test instructions?  If someone makes changes in the cdq-analysis code, they will have to rebuild their own image and add it here for testing

Reply:
Or maybe have users set an environment variable that defaults to `quay.io/redhat-appstudio/cdq-analysis:next` if unset

Reply:
updated